projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -39,6 +39,14 @@ public static function init() {
 			return;
 		}
 
+		// Edit Jetpack plugin actions link.
+		add_filter(
```

Comment:
minor. don't you think it's best if we only add the filter if Jetpack plugin is installed? Otherwise `get_installed_plugin_filename` will return null and we'll have a broken useless filter registered...

Reply:
Yes. That's make sense. Let me update it.

Reply:
Addressed [here](https://github.com/Automattic/jetpack/pull/22961/commits/13c71e2723ab1eacea0efda3a3c822ea44b20f69). 

And even more, we can polish this implementation to add My Jetpack not only in Jetpack plugin but for other plugins, when Jetpack is not installed/active or keeping in mind other conditions, too. For instance, we would add it for Boost:

<img width="322" alt="image" src="https://user-images.githubusercontent.com/77539/154579883-7989e5cd-be94-41a0-8d8d-3b6460f79488.png">

Anyway and if so, I prefer a follow-up one.